control-plane-operator/controllers/hostedcontrolplane/kcm/params.go

```diff
@@ -42,9 +42,8 @@ func NewKubeControllerManagerParams(ctx context.Context, hcp *hyperv1.HostedCont
 		FeatureGate: globalConfig.FeatureGate,
 		// TODO: Come up with sane defaults for scheduling APIServer pods
 		// Expose configuration
-		HyperkubeImage: images["hyperkube"],
-		// TODO: HCCO is the Hypershift image which contains the token-minter binary.  Messy though.
-		TokenMinterImage:        images["hosted-cluster-config-operator"],
+		HyperkubeImage:          images["hyperkube"],
+		TokenMinterImage:        images["token-minter"],
```

Comment:
we need the same change here https://github.com/openshift/hypershift/blob/9fba0b6ed55808f86b1f9d5d13d2837cf5107b5e/control-plane-operator/controllers/hostedcontrolplane/kas/params.go#L89

Reply:
Good catch, this makes me think that our use of a map and inline strings for addressing payload component identifiers is problematic. We should probably revisit this map and use constants or a more strongly typed data structure of some kind for component addressing... It wouldn't have caught this particular bug since we changed the component being referenced, but just thinking forward...

Reply:
Fixed